automation/deploy_hco.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HCO_VERSION=1.4.0
+CMD=oc
+NS=kubevirt-hyperconverged
+
+# Create a catalog source
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-unstable-catalog-source
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: "quay.io/kubevirt/hyperconverged-cluster-index:${HCO_VERSION}"
```

Comment:
quotes are not needed in image pullspec

Reply:
I prefer to use them when I'm using environment variables. Anyway, it's working and I don't see why it's worng.